### DISCUSSION TITLE
How to Add a Music Folder

### DISCUSSION THREAD

**Discussion by UPEngineer** [music-assistant/support#1162] [Q&A] [answered]
I am so lost trying to figure out how to add a music folder. In the old version of Music Assistant, it has a place to put in the folder. For example, I have NAS folders mounted to /media/musicnas/.

In the 2.0 beta of Music Assistant I can't find where to enter my information. Am I completely missing a step or can anyone tell me what I need to do? 

Thank you in advance!

Scott

**Comment by OzGav** [comment, marked as answer]
You add a  SMB provider via the settings

**Reply by UPEngineer** [reply]
Well I am an idiot. I didn't see the giant plus sign in the bottom right hand corner :)
